helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   # Do not change name, its hard-coded in operator
-  name: aerospike-cluster
+  name: aerospike-operator-controller-manager
```

Comment:
As this name is hardcoded, we get an error(SA already exists) if we create another cluster in the same namespace. Is there a way to skip the creation and deletion of SA?

Reply:
Or would it be possible that SA, Role, Rolebinding is created by operator helm for every watched namespace?

Reply:
Alright. So apparently we have to move the rbac creation to operator's helm chart as there will be conflict of ownership of the ServiceAccount (since it's hardcoded to a single name) if two AerospikeCluster(s) are deployed onto same namespace.

Reply:
Good catch @sud82